ios/browser/api/bookmarks/brave_bookmarks_api.h

```diff
@@ -73,6 +73,8 @@ OBJC_EXPORT
 @property(nonatomic, readonly) NSUInteger childCount;
 
 - (nullable IOSBookmarkNode*)childAtIndex:(NSUInteger)index;
+- (NSArray<BookmarkFolder*>*)nestedChildFoldersFiltered:(BOOL(^)(BookmarkFolder*))included
```

Comment:
I'm not following what this is supposed to do. Filtered for what? Is there a similar method for android? I definitely think this method needs some comments or a name change because it's not intuitive what is going on here or why this is needed to prevent users from moving parent folders into child folders